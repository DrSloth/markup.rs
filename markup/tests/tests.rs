use insta::assert_display_snapshot;

#[test]
fn t() {
    markup::define! {
      T1 { "Hello" }
      T2 { "Hello" ", World!" }
    }
    assert_display_snapshot!(T1 {});
    assert_display_snapshot!(T2 {});
}
