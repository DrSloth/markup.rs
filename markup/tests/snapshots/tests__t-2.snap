---
source: markup/tests/tests.rs
expression: "T2{}"
---
Hello, World!
